.github/workflows/ci.yml

```diff
@@ -10,26 +11,37 @@ on:
 
 jobs:
   main:
-    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+          os: [ubuntu-latest]
```

Comment:
we could add eg. windows and macos here, and would have a good validation, if the builds also work on other platforms. especially for development this could be interesting, eg. labeling builds only working on linux, etc.